README.md

```diff
@@ -31,23 +31,23 @@ compression, strong consistency, poison pill detection, etc.
 Message queues generally provide a loosely-coupled, asynchronous communication
 channel ("queue") between application services (producers and consumers) that
 send messages to one another. You can think about it like a mailbox for
-communication between application programs, where 'producer' drops a message in
-a mailbox and 'consumer' picks it up at its own leisure. Messages placed into
+communication between application programs, where a "producer" drops a message in
+a mailbox and a "consumer" picks it up at its own leisure. Messages placed into
 the queue are stored until the recipient retrieves and processes them. In other
 words, producer and consumer applications can temporally and spatially isolate
 themselves from each other by using a message queue to facilitate
 communication.
 
-BlazingMQ's back-end (message brokers) has been implemented in C++, and client
+BlazingMQ's back-end (message broker) has been implemented in C++, and client
```

Comment:
I'm assuming this is a mistake and that there is only a single broker, but then again I'm not really familiar with BlazingMQ and so may be off.